docs/pages/transactions/using-eip-7702/api.mdx

```diff
@@ -0,0 +1,41 @@
+See the [`wallet_prepareCalls` API
+reference](/docs/wallets/api/smart-wallets/wallet-api-endpoints/wallet-api-endpoints/wallet-prepare-calls)
```

Comment:
Should we include that you'll need the env vars, or is that outlined in the linked doc?